_15thnight/util.py

```diff
@@ -0,0 +1,44 @@
+import simplejson as json
```

Comment:
simplejson was needed for python < 2.6. Are we trying to support really old python? If not, all we need is python's built in json[0] OR we can use Flask's json[1] which handles all that json/simplejson problem for us.

[0] https://docs.python.org/2/library/json.html
[1] http://flask.pocoo.org/docs/0.11/api/#module-flask.json